scipy/stats/_multivariate.py

```diff
@@ -5057,6 +5059,175 @@ def rvs(self, size=1, random_state=None):
                               size=size,
                               random_state=random_state)
 
+class dirichlet_multinomial_gen(multi_rv_generic):
+    r"""A Dirichlet multinomial random variable.
+
+    Methods
+    -------
+    logpmf(alpha, x, n):
+        Log of the probability mass function.
+    pmf(alpha, x, n):
+        Probability mass function.
+    mean(alpha, n):
+        Mean of the Dirichlet multinomial distribution.
+    var(alpha, n):
+        Variance of the Dirichlet multinomial distribution.
+    cov(alpha, n):
+        The covariance of the Dirichlet multinomial distribution.
+
+    Parameters
+    ----------
+    %(_dirichlet_doc_default_callparams)s
+    %(_doc_random_state)s
+
+    References
+    ----------
+    .. [1] Dirichlet-multinomial distribution,
+     https://www.wikipedia.org/wiki/Dirichlet-multinomial_distribution
+     """
+    def __init__(self, seed=None):
+        super().__init__(seed)
+        self.__doc__ = doccer.docformat(self.__doc__, dirichlet_docdict_params)
+
+    def __call__(self, alpha, seed=None):
+        return dirichlet_frozen(alpha, seed=seed)
```

Comment:
Create a `dirichlet_multinomial_frozen` class, instantiate it here, and test it.

Reply:
Done.